OneCommunityGlobal/HGNRest
# Anirudh - Fetch end date field from dashboardhelper

# Description
Add to Leaderboard a way to see how many days until a person’s last day. A scheduled last day can be set in 3 places:

1. User Management>Set Final Day
2. Profile>Basic>Set Final Day Information
3. Profile>Volunteer Times tab>End Date

## Related PRS (if any):
This backend PR is related to the frontend PR [#2220](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2220)
…

## Main changes explained:
After this Date has been set, you can see the count of the remaining days on the leaderboard, under the days left column.
I have also fixed `Unsafe arithmetic operation on optional chaining` errors in `dashboardhelper.js`

## How to test:

1. check into current branch
2. do npm install and npm run build && npm run start to run this PR locally
3. Follow the instructions on the frontend PR
